controllers/rcj_soccer_referee_supervisor/rcj_soccer_referee_supervisor.py

```diff
@@ -24,11 +26,12 @@ def reflog_path(
     if not directory.exists():
         directory.mkdir(parents=True, exist_ok=True)
 
-    p = directory / Path(f'{team_blue}_vs_{team_yellow}-{now_str}.jsonl')
+    p = directory / Path(f'{team_blue}_vs_{team_yellow}-{now_str}')
     return p
 
 
-reflog = reflog_path(Path('reflog'), TEAM_BLUE, TEAM_YELLOW)
+reflog = output_path(Path('reflog'), TEAM_BLUE, TEAM_YELLOW).with_suffix('.jsonl')
```

Comment:
@aquahika I like this very much!

Just one small nitpick: I think it would be a bit better to factor-out the `reflog` path, so that we don't end up we two separate filenames by some random chance  (we'd be calling `datetime.utcnow().strftime('%Y%m%dT%H%M%S.%fZ')` twice and that could end up being different between two ticks of Python's event loop).

Something like the following might work:

```python
output_prefix = output_path(Path('reflog'), TEAM_BLUE, TEAM_YELLOW)
reflog_path = output_prefix.with_suffix('.jsonl')
video_path =  output_prefix.with_suffix('.mp4')

```

Reply:
Please, use `consts.Team.BLUE.value` and `consts.Team.YELLOW.value`

Reply:
@Adman that's different though, right?

`consts.Team.BLUE.value` and `consts.Team.YELLOW.value` will always be `'B'` and `'Y'` whereas `TEAM_BLUE` and `TEAM_YELLOW`  will hopefully get set to the respective team names somehow (via command line arguments and/or env variables?)

Reply:
Certainly it is. I'll change it.

Reply:
> Please, use consts.Team.BLUE.value and consts.Team.YELLOW.value

This is coming from a recent `master` branch, so let us make it at another branch/PR if needed ;)